src/main/java/models/Account.java

```diff
@@ -0,0 +1,56 @@
+package models;
+
+public class Account {
+
+    private String email;
+    private String password;
+    private String confirmPassword;
```

Comment:
this is not a valid property for Account class

Reply:
Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.